src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -1269,7 +1260,7 @@ class CaseExpressionSegment(BaseSegment):
         ),
     ),
     # Expression_B_Grammar https://www.cockroachlabs.com/docs/v20.2/sql-grammar.htm#b_expr
-    Expression_B_Grammar=None,  # TODO
+    # Expression_B_Grammar=None,  # TODO
```

Comment:
Why is this commented out?

Reply:
MyPy was complaining that `None` couldn't be in a dialect.

Reply:
So the code still works okay? That was my main concern.

Re: The MyPy warning, can we (should we?) change a type hint, e.g. `Optional[Thing]` rather than just `Thing`?

Reply:
Yeah the code works fine. `Expression_B_Grammar` was never implemented and nothing refers to it. It's a remnant from when the expression structure was much closer to the cockroach one. We never actually needed it though. There's an argument for actually pulling it out altogether rather than just commenting it out - do you think that would be tidier?

Reply:
Interesting... since several comments in the ANSI dialect expression grammar refer to corresponding parts of the Cockroach one, I'm in favor of keeping this line in place (but commented out as you have it here). I suggest adding a comment similar to what you said above, e.g.:
>Expression_B_Grammar is an unused remnant from CockroachDB, whose expression grammar was the original basis for SQLFluff's.

Reply:
I've updated the comment with some new wording to explain what's going on with Expression B.

> CockroachDB defines Expression_B_Grammar. The SQLFluff implementation of
> expression parsing pulls that logic into Expression_A_Grammar and so there's
> currently no need to define Expression_B.